static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx

```diff
@@ -141,6 +150,7 @@ class IssueWidgetQueries extends React.Component<Props, State> {
         'issue.id': id,
         issue: shortId,
         title,
+        project: project?.slug,
```

Comment:
can project be `undefined/null `here? if yes, should we update the types?

Reply:
project should be defined in Group as far as I can tell. I haven't seen a case where it hasn't been defined yet. I've updated this to just `project.slug` now. Thanks!